lms/static/scripts/frontend_apps/utils/dashboard/root-view-title.ts

```diff
@@ -0,0 +1,10 @@
+import type { DashboardConfig } from '../../config';
+
+/**
+ * Resolve the title for the dashboard's root view.
+ * That is the organization name if teh dashboard was launched for one specific
+ * org, or "All courses" otherwise.
```

Comment:
```suggestion
 * Return the title for the dashboard's root view.
 *
 * This is the organization name if the dashboard was launched for one specific
 * org, or "All courses" otherwise.
```

"Resolve" is OK, but to me suggests more expensive work than is being done.